src/test/java/cloud/fogbow/ras/core/OrderStateTransitionerTest.java

```diff
@@ -70,6 +69,49 @@ public void testTransitionToChangeOrderStateOpenToSpawning() throws UnexpectedEx
         Assert.assertNull(openOrdersList.getNext());
     }
 
+    // test case: When calling the transition() method, it must change the state of Order
+    // from the remote order passed as parameter to a specific OrderState. This Order
+    // defined originally with Open, will be changed to the Spawning state, removed
+    // from the open orders list, and added to the spawning orders list.
+    @Test
+    public void testTransitionToChangeOrderStateOpenToSpawningWithRemoteProvider() throws Exception {
+
+        // set up
+        OrderState originState = OrderState.OPEN;
+        OrderState destinationState = OrderState.SPAWNING;
+
+        this.testUtils.mockReadOrdersFromDataBase();
+
+        SharedOrderHolders orderHolders = SharedOrderHolders.getInstance();
+
+        SynchronizedDoublyLinkedList<Order> openOrdersList = orderHolders.getOpenOrdersList();
+        SynchronizedDoublyLinkedList<Order> spawningOrdersList = orderHolders.getSpawningOrdersList();
+
+        Order order = createOrder(originState);
+        String remoteMember = "fake-remote-member";
+        order.setRequester(remoteMember);
```

Comment:
You can use the FAKE_REMOTE_MEMBER_ID constant from class 'TestUtils' for test standardization only.

Reply:
I merged with develop and did those changes.